packages/web-app-files/src/components/SideBar/Details/FileDetails.vue

```diff
@@ -78,6 +78,70 @@
             />
           </td>
         </tr>
+        <tr v-if="runningOnEos">
+          <th scope="col" class="oc-pr-s" v-text="eosPathLabel" />
+          <td>
+            <div class="oc-flex oc-flex-middle oc-flex-between oc-width-1-1">
+              <div class="oc-flex oc-flex-middle">
+                <p
+                  ref="filePath"
+                  v-oc-tooltip="file.path"
+                  class="oc-my-rm oc-text-truncate"
+                  v-text="file.path"
+                />
+              </div>
+              <oc-button
+                oc-tooltip="Copy EOS path"
+                appearance="raw"
+                aria-label="Copy EOS path"
+                class="oc-files-private-link-copy-url"
+                :variation="copiedEos ? 'success' : 'passive'"
+                @click="copyEosPathToClipboard"
+              >
+                <span text="EOS Path" />
+                <oc-icon
+                  v-if="copiedEos"
+                  key="oc-copy-to-clipboard-copied"
+                  name="checkbox-circle"
+                  class="_clipboard-success-animation"
+                />
+                <oc-icon v-else key="oc-copy-to-clipboard-copy" name="clipboard" />
+              </oc-button>
+            </div>
+          </td>
+        </tr>
+        <tr>
+          <th scope="col" class="oc-pr-s" v-text="directLinkLabel" />
+          <td>
+            <div class="oc-flex oc-flex-middle oc-flex-between oc-width-1-1">
+              <div class="oc-flex oc-flex-middle">
+                <p
+                  ref="directLink"
+                  v-oc-tooltip="directLink"
+                  class="oc-my-rm oc-text-truncate"
+                  v-text="directLink"
+                />
+              </div>
+              <oc-button
+                oc-tooltip="Copy direct link"
+                appearance="raw"
+                aria-label="Copy direct link"
+                class="oc-files-private-link-copy-url"
+                :variation="copiedDirect ? 'success' : 'passive'"
+                @click="copyDirectLinkToClipboard"
+              >
+                <span text="EOS Path" />
```

Comment:
Please use a `label` prop and `this.$gettext()` here

Reply:
Actually already exists as `eosPathLabel`, just needs to be used

Reply:
I'm thinking I should remove these (this and line 101). They are not showing because `text` is not a valid attribute so they are effectively empty. If I make them have something inside, we get this:

![image](https://user-images.githubusercontent.com/6058151/168782825-92d2583f-15fd-4ae1-bac7-e8aa48d584e4.png)

I assume there is no reason to have text there, but correct me if I'm wrong (pinging @diocas).

Reply:
You could use `v-text` and a computed property to show text (`text` itself won't work), but agreed that you could use an icon for "visual" usage and then both aria-label and tooltip on the wrapping button for contextual/screenreader help